Tests/Functional/MappingToElasticaTest.php

```diff
@@ -21,11 +21,13 @@ class MappingToElasticaTest extends WebTestCase
     public function testResetIndexAddsMappings()
     {
         $client = $this->createClient(array('test_case' => 'Basic'));
-        $resetter = $this->getResetter($client);
-        $resetter->resetIndex('index');
 
         $type = $this->getType($client);
         $mapping = $type->getMapping();
+        $resetter = $this->getResetter($client);
```

Comment:
However, a lot of other things have been deprecated in ES2.x, so other errors will come up after that.